internal/tests/api/nodeset_test.go

```diff
@@ -48,6 +40,11 @@ func TestNodeSetRegistration_NoRegistration(t *testing.T) {
 	require.NoError(t, err)
 	t.Log("Recover called")
 
+	nodesetTestWalletRecoveredSnapshot, err = testMgr.CreateSnapshot()
```

Comment:
In the future we should roll the wallet integration tests in `wallet_test.go` into these (and honestly just make them all under the same package now that snapshotting is a thing) because those will end up duplicating this here.

Reply:
done